packages/hapi/modules/plugin/plugin.js

```diff
@@ -37,13 +37,13 @@ const createPlugin = ({ options } = {}) => {
   const plugin = {
     name: pkg.name,
     version: pkg.version,
-    register(server) {
+    register(server, o, done) {
       server.ext('onRequest', (request, h) => {
         request.promster = { start: process.hrtime() };
         return h.continue;
       });
 
-      server.events.on('response', request => {
```

Comment:
I added a version check in later versions. Please check master if this does the trick for you.

Reply:
good catch. updated PR with checking on function done().